src/OpenTelemetry.Api/.publicApi/Experimental/net8.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,80 @@
+abstract OpenTelemetry.Logs.Logger.EmitLog(in OpenTelemetry.Logs.LogRecordData data, in OpenTelemetry.Logs.LogRecordAttributeList attributes) -> void
```

Comment:
Across this PR all of this stuff should go into the **shipped** files. These APIs have all shipped we don't want to allow them to be modified for net8.0 anymore than we want to allow them to be modified for other targets 😄

When we go to do the public api review for 1.7.0 what we need to see in the "unshipped" files is new stuff. This is going to add too much noise.

Reply:
Woops looks like it is correct... this is an experimental one. Sorry @vishweshbankwar!